lti/introduction.rst

```diff
@@ -103,39 +166,14 @@ with the "unsupported" message defined in the LTI specification.
 
 .. warning::
 
-  LTI exercises do not currently support group submissions. Students may only
-  submit as individuals. Group submissions could be implemented
-  for LTI exercises in the future.
-
-.. warning::
-
-  A submission to an LTI exercise can not be sent to the exercise service for
-  (automatic) regrading. (Normal exercises could be regraded from the teacher's
-  inspect submission page.) The LTI protocol does not support any kind regrading
-  initiated by the LMS. However, if the Tool Provider stores the submission data,
-  then it could provide its own regrading functionality that would also send the
-  new results to the LMS via normal replaceResult requests. It depends on the
-  TP whether such functionality is supported. Remember though that the
-  replaceResult requests do not overwrite the previous submissions already
-  stored in A+.
-  
-  **N.B.** A+ stores the submission history and uses the best submissions for
-  computing the total points in the course. If an exercise has been faulty and
-  submissions have earned too many points, the submissions must be reset by hand
-  in order to remove the incorrect points since LTI exercises do not support
-  automatic regrading.
-
-.. warning::
-
-  LTI exercises have limitations for tracking the number of submissions and
-  the submission time. This is relevant when strict deadlines and submission
-  limits are used. Since the Basic Outcomes Service does not explicitly include
-  these data fields, A+ can not track them completely accurately. If the Tool
-  Provider sends a replaceResult request to A+ for each submission and the
-  requests are sent quickly without delay at the time the student uploads the
-  submission, then A+ may track the correct number of submissions and the
-  recorded submission times do not differ much from the real submission times.
-
-
-.. _the official standard: https://www.imsglobal.org/specs/ltiv1p1/implementation-guide
-
+  As with LTI 1.3, LTI 1.1 exercises have limitations for tracking the number of
+  submissions and the submission time. This is relevant when strict deadlines
+  and submission limits are used. Since the Basic Outcomes Service does not
+  explicitly include these data fields, A+ can not track them completely
+  accurately. If the Tool Provider sends a replaceResult request to A+ for each
+  submission and the requests are sent quickly without delay at the time the
+  student uploads the submission, then A+ may track the correct number of
```

Comment:
Improved sentence:
...the requests are sent quickly without _**a**_ delay _**when**_ the student uploads the submission...